fluss-server/src/main/java/com/alibaba/fluss/server/kv/snapshot/KvTabletSnapshotTarget.java

```diff
@@ -219,14 +219,9 @@ public void handleSnapshotResult(
         } catch (Exception e) {
             Throwable t = ExceptionUtils.stripExecutionException(e);
             snapshotsCleaner.cleanSnapshot(completedSnapshot, () -> {}, ioExecutor);
-            handleSnapshotFailure(
-                    snapshotId,
-                    snapshotLocation,
-                    new FlussException(
-                            String.format(
-                                    "Fail to add the snapshot %s to completed snapshot store.",
-                                    completedSnapshot),
-                            t));
+            handleSnapshotFailure(snapshotId, snapshotLocation, t);
+            // throw the exception to make PeriodicSnapshotManager can catch the exception
```

Comment:
Throw the exception so that `PeriodicSnapshotManager` can warn `Fail to handle snapshot result during snapshot  ..`